ui/admin/app/templates/scopes/scope/aliases/index.hbs

```diff
@@ -48,11 +48,11 @@
                   @route='scopes.scope.aliases.alias'
                   @model={{B.data.alias.id}}
                 >
-                  {{B.data.alias.displayName}}
+                  {{B.data.alias.value}}
```

Comment:
this should be value instead of name

Reply:
this is based on design/product [feedback](https://hashicorp.slack.com/archives/C012GTH1C4E/p1713560858991069). Value is a required field and we show that in the table as it makes more sense than ID for aliases